gormschema/gorm_test.go

```diff
@@ -3,20 +3,105 @@ package gormschema
 import (
 	"testing"
 
+	"ariga.io/atlas-go-sdk/recordriver"
+	ckModels "ariga.io/atlas-provider-gorm/internal/testdata/circular_fk_models"
 	"ariga.io/atlas-provider-gorm/internal/testdata/models"
 	"github.com/stretchr/testify/require"
 	"gorm.io/gorm"
 )
 
-func TestConfig(t *testing.T) {
-	l := New("sqlite", WithConfig(
+func TestSQLiteConfig(t *testing.T) {
+	l := New("sqlite")
+	sql, err := l.Load(models.Pet{}, models.User{}, ckModels.Event{}, ckModels.Location{})
+	require.NoError(t, err)
+	require.Contains(t, sql, "CREATE TABLE `pets`")
+	require.Contains(t, sql, "CREATE TABLE `users`")
+	require.Contains(t, sql, "CREATE TABLE `events`")
+	require.Contains(t, sql, "CREATE TABLE `locations`")
+	require.Contains(t, sql, "FOREIGN KEY (`eventId`)")
+	require.Contains(t, sql, "FOREIGN KEY (`locationId`)")
+	require.Contains(t, sql, "CONSTRAINT `fk_users_pets` FOREIGN KEY")
```

Comment:
Use equal comparison (and not contains) as the order of the statements does matter.